shopinvader/models/res_partner.py

```diff
@@ -43,6 +43,9 @@ class ResPartner(models.Model):
         compute_sudo=True,
         store=True,
     )
+    has_shopinvader_user_active = fields.Boolean(
```

Comment:
this field should be computed as well otherwise it won't have any value and your query won't be useful at all

Reply:
but is useless in you SQL if it's not populated... I don't get your point, sorry :)

Reply:
the compute method is present and stayed in shopinvader_customer_validate
https://github.com/shopinvader/odoo-shopinvader/blob/220d9467eb13af5ea2fe37664e13dc8af65bfc30/shopinvader_customer_validate/models/res_partner.py#L18

Reply:
ok, let's put it this way: what's the role of `has_shopinvader_user_active` here when you don't have `s_c_validate` installed?

Reply:
none, if we don't have s_c_validate, has_shopinvader_user_active will have false value and behavior will be the same as before
I don't want to bring compute method in shopinvader as then I will need to bring more code regards partner states here
maybe a more graceful option is to add an option via a hook in constrain statement  and in s_c_validate return the proper condition